cmd/podman/images/save.go

```diff
@@ -91,11 +96,33 @@ func save(cmd *cobra.Command, args []string) error {
 		return errors.Errorf("--compress can only be set when --format is either 'oci-dir' or 'docker-dir'")
 	}
 	if len(saveOpts.Output) == 0 {
-		fi := os.Stdout
-		if terminal.IsTerminal(int(fi.Fd())) {
-			return errors.Errorf("refusing to save to terminal. Use -o flag or redirect")
+		pipeFile := "/tmp/saveio"
```

Comment:
This must not use a predictable name; generate a random one similarly to `mktemp`.

… and there probably isn’t a helper to do that in the standard library. 😢 Using `ioutil.TempFile`+`os.Remove`+`Mkfifo` would not be safe.

Reply:
I don't think we should delete an existing file as it could be used by another podman instance.  Wouldn't it be enough to generate a random name and try again on `EEXIST`?

Reply:
One standard pattern is to `mktemp` a directory, not a file, and create the pipe/socket/whatever therein. Implementation and cleanup are left as an exercise for the reader.

Reply:
Oh, that’s a great trick! Good idea.